Source/FortranInterface/WarpX_picsar.F90

```diff
@@ -301,6 +314,58 @@ subroutine warpx_current_deposition( &
 
   end subroutine warpx_current_deposition
 
+  ! _________________________________________________________________
+  !>
+  !> @brief
+  !> Applies the inverse volume scaling for RZ current deposition
+  !>
+  !> @details
+  !> The scaling is done for both single mode (FDTD) and
+  !> multi mode (spectral)
```

Comment:
The above comment might be confusing. 
My understanding is that a rz PIC code can be:
- FDTD with a single azimuthal mode (e.g. WarpX with the current PR)
- FDTD with multiple azimuthal modes (e.g. Warp)
- Spectral with a single azimuthal mode (e.g. FBPIC with `Nm=1`)
- Spectral with multiple azimuthal modes (e.g. FBPIC with `Nm>1`)

(where, here, "Spectral" means using the PSATD algorithm with a Fourier transform in z and Hankel transform in r)

Therefore, I would suggest to remove `(FDTD)` and `(spectral)` in the above comment.